mackerel-plugin-linux/lib/linux.go

```diff
@@ -331,7 +331,7 @@ func collectDiskStats(path string, p *map[string]interface{}) error {
 
 func parseDiskStat(name, stat string, p *map[string]interface{}) error {
 	fields := strings.Fields(stat)
-	if len(fields) != 11 {
+	if len(fields) < 11 {
```

Comment:
This is off topic but how about adding a comment for this magic number?

Reply:
I don't think this as magic number anymore, since this requirement is just for accessing `fields[10]` safely at L340.
(Agree that this *was* a magic number!)

Reply:
Ah, I've noticed L340 says what the magic number was. Thanks.